config/settings/base.py

```diff
@@ -538,3 +540,10 @@
     MIDDLEWARE.append(ADSERVER_IPADDRESS_MIDDLEWARE)
 if ADSERVER_GEOIP_MIDDLEWARE:
     MIDDLEWARE.append(ADSERVER_GEOIP_MIDDLEWARE)
+
+
+CORS_ALLOWED_ORIGINS = [
```

Comment:
This looks like a development setting. Perhaps make the prod setting read from env?